src/hiccup/compiler.clj

```diff
@@ -23,10 +23,15 @@
   (if (xml-mode?) " />" ">"))
 
 (defn- render-style-map [value]
-  (->> value
-       (map (fn [[k v]] (str (util/as-str k) ":" (util/as-str v) ";")))
-       (sort)
-       (apply str)))
+  (let [^StringBuilder sb (StringBuilder.)]
+    (util/fast-run
+      (fn [[k v]]
+        (.append sb (util/to-str k))
+        (.append sb ":")
+        (.append sb (util/to-str v))
+        (.append sb ";"))
```

Comment:
Hiccup formats according to the Clojure Style Guide, so functions should be formatted like:

```clojure
(f
 a
 b)
```

Rather than:

```clojure
(f
  a
  b)
```

Reply:
(Also lines should be within 80 characters in length, ideally.)

Reply:
52d7bfb0a652851cea1adc796fc9d1320fabfcdc